### ISSUE TITLE
Rt crowd forecast: Population adjustment

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/europe-covid-forecast#49]
In the Rt model the R estimate has population adjustment beyond the forecast horizon. This means that what you see for Rt in the forecast will actually slightly reduce as susceptibility reduces over time. There are two options. 1. Remove the control parameter so this is not the case (this will induce a divergence with the normal Rt model or 2. flag this to users.

**Comment by seabbs** [comment]
No, it doesn't hence mentioning it as if it did as you say not a problem.

**Comment by nikosbosse** [comment]
Intuitively I'd have a small preference for removing the control parameter (if we think it makes a meaningful difference). I'm not sure people will be able to understand what it does and what effect it has and it would be easiest if the app was as close as posisble to WYSIWYG. 
Do you think the divergence in methodology would pose a problem? Maybe should have a quick chat about this

**Comment by seabbs** [comment]
its the pop arg supplied to `rt_obs` and it needs to be updated in the Rt fit model object prior to use in the app but after use for forecasting Rt via the model.
